receiver/azureeventhubreceiver/config.go

```diff
@@ -66,3 +66,4 @@ func (config *Config) Validate() error {
 	}
 	return nil
 }
+// TODO(Sambhav800): Consider adding more configuration options.
```

Comment:
Can you explain how and whom this helps? In my view, the comment is vague and not helpful.